config/settings.yml

```diff
@@ -13,6 +13,7 @@ kafka_producer_topics:
   upload_validation: 'platform.upload.validation'
   payload_tracker: 'platform.payload-status'
   remediation_updates: 'platform.remediation-updates.compliance'
+  test: 'test'
```

Comment:
Is this Kafka topic always available?

Reply:
Do topics have to be 'available'? What does that mean? afaik, any topic can be produced/consumed, it just requires two services to agree on the topic name, but idk for sure.

Reply:
Me neither. It might end up in the `LeaderNotAvailable` as noted below when used for the first time.